lib/pbench/test/unit/server/test_requests.py

```diff
@@ -265,22 +243,16 @@ def test_bad_extension_upload(
         assert not self.cachemanager_created
 
     def test_invalid_authorization_upload(
-        self, client, caplog, server_config, setup_ctrl, pbench_token
+        self, client, caplog, server_config, setup_ctrl, pbench_token_invalid
     ):
-        # Log out the user
-        response = client.post(
-            f"{server_config.rest_uri}/logout",
-            headers={"Authorization": "Bearer " + pbench_token},
-        )
-
         # Upload with invalid token
         response = client.put(
             self.gen_uri(server_config),
-            headers=self.gen_headers(pbench_token, "md5sum"),
+            headers=self.gen_headers(pbench_token_invalid, "md5sum"),
         )
         assert response.status_code == HTTPStatus.UNAUTHORIZED
         for record in caplog.records:
-            assert record.levelname in ["DEBUG", "INFO"]
+            assert record.levelname in ["DEBUG", "INFO", "ERROR"]
```

Comment:
We should never have `ERROR` log output unless something's broken inside the server code. "User" errors are at worst `WARNING` although we should be logging those sparingly and only in very unusual circumstances.